ansible/playbook/k8s_extras.yml

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
Such playbooks `k8s.yml, k8s-misc.yml` already exist under the path `kubernetes/ansible/`. So why do you still use it here ?